src/support/allocators/secure.h

```diff
@@ -53,6 +47,17 @@ struct secure_allocator : public std::allocator<T> {
         }
         LockedPoolManager::Instance().free(p);
     }
+
+    template <typename U>
+    friend bool operator==(const secure_allocator&, const secure_allocator<U>&) noexcept
+    {
+        return true;
+    }
+    template <typename U>
+    friend bool operator!=(const secure_allocator&, const secure_allocator<U>&) noexcept
+    {
+        return true;
```

Comment:
This look likes a copy-pasta.
```suggestion
        return false;
```

Reply:
Thanks!